internal/ask/form/submission/submission.go

```diff
@@ -107,7 +107,7 @@ type Answer struct {
 	Identity     bool        `json:"identity" bson:"identity"`
 	Answer       interface{} `json:"answer" bson:"answer"`
```

Comment:
The front end uses complex looking objects to describe answers. We require coordination with the front end to map out concrete types for the answers being used here.